app/helpers/device_stories_helper.rb

```diff
@@ -72,4 +72,34 @@ def last_air_quality_values(last_values)
       last_values[0..index_u - 1]
     end
   end
+
+  CATEGORY_SENSOR_NAMES = {
+    'radiation_sensors' => %w(lnd_7128ec lnd_7318c lnd_712u lnd_7318u lnd_78017w lnd7318u lnd7128c),
+    'air_sensors' => %w(pms_pm10_0 pms_pm02_5 pms_pm01_0),
+    'bat_voltage' => ['bat_voltage'],
+    'temperature_C' => ['temperature_C'],
+    'temperature_F' => ['temperature_F'],
+    'humidity' => ['humidity'],
+    'pressure' => ['pressure'],
+    'charging' => ['charging']
+  }.freeze
+
+  def sensor_data_by_sensors(sensor_data)
+    data_by_dates = convert_by_dates(sensor_data.dig('aggregations', 'sensor_data', 'buckets') || [])
+    CATEGORY_SENSOR_NAMES.transform_values do |names|
+      names.each_with_object([]) do |name, arr|
+        arr << { name: name, value: data_by_dates[name] } if data_by_dates.key?(name)
+      end
+    end
+  end
+
+  def convert_by_dates(buckets)
+    buckets.each_with_object(Hash.new { |h, k| h[k] = {} }) do |bucket, hsh|
+      date = bucket['key_as_string']
```

Comment:
This line expects data from 'IngestMeasurement.query_sensor_data` in that `key_as_string` is formatted in `yyyy-MM-dd HH`.